### PR TITLE
GEOPY-835

### PR DESCRIPTION
**Pull request by domfournier**
**GEOPY-835 - Iso-surface creation jupyter-app error after crash of marhcing cube**
…o float parameter

### CODE REVIEW

File: geoapps/iso_surfaces/driver.py [hunk 1]
```diff
@@ -158,9 +160,13 @@ def iso_surface(
...
             values = values[0].reshape(x.shape)
 
+        lower, upper = np.nanmin(values), np.nanmax(values)
         surfaces = []
-        for level in levels:
+        print("Running marching cubes on levels.")
+        for level in tqdm(levels):
             try:
+                if level < lower or level > upper:
+                    continue
```
Comment (benk-mira) [inline geoapps/iso_surfaces/driver.py:169]: Maybe log that the levels are being skipped?

Comment (domfournier) [reply geoapps/iso_surfaces/driver.py:169]: Good call. Done

Comment (benk-mira) [reply geoapps/iso_surfaces/driver.py:172]: That should do it!

**Review by benk-mira — APPROVED** [review]
cool

**Pull request merged**